airbyte-ci/connectors/pipelines/pipelines/publish_pypi.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from textwrap import dedent
+from typing import List
+
+import anyio
+from dagger import ExecError
+from pipelines.bases import ConnectorReport, Step, StepResult, StepStatus
+from pipelines.contexts import PublishConnectorContext
+from pipelines.pipelines import metadata
+
+
+class PublishPyPIConnector(Step):
+    title = "Publish connector to PyPI"
+
+    async def _run(self) -> StepResult:
+        if self.context.connector.language not in ["python", "low-code"]:
+            return self.skip("Only Python connectors can be published to PyPI.")
+
+        try:
+            setup_cfg = dedent(
+                f"""
+                [metadata]
+                name = airbyte-{self.context.connector.technical_name}
```

Comment:
Does this mean the name of the package will be `airbyte-source-salesforce` ? I think that's fine, small note here that the module is still named `source_salesforce` (without the prefix). Probably not an issue, just calling it out